dev/io.openliberty.jakarta.activation.2.0/.classpath

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Shouldn't the removal of the com.ibm.websphere.jakarta.activation.2.0 component be deleted as part of this PR?